CTP/doc/shell/shell_guide.md

```diff
@@ -0,0 +1,667 @@
+# Shell Test Guide
+# 1 Test Introduction
+Shell test is an important test suit in cubrid test.  
+It contains almost all the feature test and some performance test which cannot be test by sql test or other test suits.  
+
+# 2 Tools Introduction
+CTP is the only test tool which is used in shell test.   

```

Comment:
Please introduce overall shell usage based on CTP.
1. test configuration
2. how to execute : 
ctp.sh shell -c shell.conf
3. find test results
4. move ```9 excluded list``` to here chapter

Reply:
TBD